wrappers/python/simtk/openmm/app/topology.py

```diff
@@ -404,3 +436,32 @@ def __init__(self, name, element, index, residue, id):
 
     def __repr__(self):
         return "<Atom %d (%s) of chain %d residue %d (%s)>" % (self.index, self.name, self.residue.chain.index, self.residue.index, self.residue.name)
+
+class Bond(tuple):
```

Comment:
I gave some example code in #1443 that used a namedtuple. You can use it the same way you used tuple here, except it also lets you access the two members via attributes like atom1 and atom2